app/actions/handlers.py

```diff
@@ -192,7 +179,7 @@ async def action_pull_events(integration, action_config: PullEventsConfig):
                             config=action_config
                         )
                     )
-                    integrated_alerts = await integrated_alerts_task
+                    integrated_alerts, dataset_status, dataset_metadata = await integrated_alerts_task
```

Comment:
Suggestion: I saw in a couple of places that you use `asyncio.create_task()` to trigger `client.get_integrated_alerts` and then you `await` the task immediately after. Then why not awaiting the coroutine `client.get_integrated_alerts` directly?